pkg/scheduler/clustereligibilitychecker/checker.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+// Package clustereligibilitychecker features a utility for verifying if a member cluster is
+// eligible for resource placement.
+package clustereligibilitychecker
```

Comment:
```suggestion
package clustereligibilityvalidator
```

niiit, how about using validator?

Reply:
I have some reservations; `validator` assumes that we are verifying the validity of the eligibility, which is not the true purpose of this package.